test/functional/feature_nulldummy.py

```diff
@@ -77,7 +78,7 @@ def run_test(self):
         cms = self.nodes[0].createmultisig(1, [self.pubkey])
         wms = self.nodes[0].createmultisig(1, [self.pubkey], 'p2sh-segwit')
         self.ms_address = cms["address"]
-        ms_unlock_details = {"scriptPubKey": self.nodes[0].validateaddress(self.ms_address)["scriptPubKey"],
+        ms_unlock_details = {"scriptPubKey":address_to_scriptpubkey(self.ms_address).hex(),
```

Comment:
nit: should be a space here

```suggestion
        ms_unlock_details = {"scriptPubKey": address_to_scriptpubkey(self.ms_address).hex(),
```

Reply:
Thanks